changelogs/fragments/705-support-boot-disk-resizing-renaming.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+      - vm_infra - Support boot disk renaming and resizing. (https://github.com/oVirt/ovirt-ansible-collection/pull/705)
```

Comment:
Adding new parameters seems to me like a RFE, so it should be a major change. Am I right @mnecas ?

Reply:
@mnecas